### ISSUE TITLE
Pickup service endpoint hardcoded

### ISSUE THREAD

**Issue by bramvankooten** [AreaFiftyLAN/lancie-api#519]
The endpoint that retrieves all the tickets that bought the Pickup Service is now hardcoded for the TicketOption with name `pickupService`

**Comment by Fastjur** [comment]
I don't think this issue is closed, the connected MR was never merged, it was closed.

**Comment by Fastjur** [comment]
Or is this related to https://github.com/AreaFiftyLAN/lancie-api/issues/526 as in, we will not use the pickup service endpoint anymore ever?
